app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue

```diff
@@ -108,26 +108,43 @@
           <div v-if="isEmailOrWebWidgetInbox">
             <label>
               {{ $t('NEW_CONVERSATION.FORM.MESSAGE.LABEL') }}
-              <reply-email-head
-                v-if="isAnEmailInbox"
-                :cc-emails.sync="ccEmails"
-                :bcc-emails.sync="bccEmails"
-              />
-              <label class="editor-wrap">
```

Comment:
Moved out the label because it caused some weird click issues when we clicked the editor placeholder text.

**Screen recordings**

https://github.com/chatwoot/chatwoot/assets/64252451/a960b699-1885-403e-989d-df4e82f9bf2e